src/main/scala/it/pagopa/interop/agreementprocess/common/Adapters.scala

```diff
@@ -396,4 +400,23 @@ object Adapters {
         case _                                            => InvalidAttributeStructure.asLeft
       }
   }
+
+  implicit class InstitutionWrapper(private val ist: SelfcareV2Dependency.Institution) extends AnyVal {
+    def toApi: Either[Throwable, Institution] = for {
+      id             <- ist.id.toRight(SelfcareEntityNotFilled(ist.getClass().getName(), "id"))
+      digitalAddress <- ist.digitalAddress.toRight(SelfcareEntityNotFilled(ist.getClass().getName(), "digitalAddress"))
+      description    <- ist.description.toRight(SelfcareEntityNotFilled(ist.getClass().getName(), "description"))
+    } yield (Institution(id = id, digitalAddress = digitalAddress, description = description))
+  }
+
+  implicit class UserResponseWrapper(private val ur: SelfcareV2Dependency.UserResponse) extends AnyVal {
+    def toApi: Either[Throwable, UserResponse] = for {
+      id      <- ur.id.toRight(SelfcareEntityNotFilled(ur.getClass().getName(), "id"))
+      uuid    <- id.toUUID.toEither
+      name    <- ur.name.toRight(SelfcareEntityNotFilled(ur.getClass().getName(), "name"))
+      surname <- ur.surname.toRight(SelfcareEntityNotFilled(ur.getClass().getName(), "surname"))
+      taxCode <- ur.taxCode.toRight(SelfcareEntityNotFilled(ur.getClass().getName(), "taxCode"))
+      email   <- ur.email.toRight(SelfcareEntityNotFilled(ur.getClass().getName(), "email"))
```

Comment:
it seems email is not used, we can ignore it to avoid errors on unnecessary field
```suggestion
```